### PR TITLE
WE-7216 create email password

### PR DESCRIPTION
**Pull request by walsh9**
Flow for creating a password from fb-only account
needs updates from nypr-account-settings and publisher

### CODE REVIEW

File: app/controllers/profile.js [hunk 1]
```diff
@@ -49,6 +49,28 @@ export default Controller.extend({
     });
   },
 
+  requestTempPassword(email) {
+    console.log('email', email);
```
Comment (noslouch) [inline app/controllers/profile.js]: let's get rid of this log

File: app/router.js [hunk 2]
```diff
@@ -68,6 +68,7 @@ Router.map(function() {
   // from the django site. It will be used when there's nothing more
   // specific.
   this.route('djangorendered', { path: '*upstream_url' });
+  this.route('set-password');
```
Comment (noslouch) [inline app/router.js]: you should add new routes above the catch all

File: tests/acceptance/login-test.js [hunk 3]
```diff
@@ -113,15 +113,15 @@ test('Log in with Facebook button is visible at load', function(assert) {
...
+  let user = server.create('user', 'facebook');
   registerMockOnInstance(this.application, 'torii-provider:facebook-connect', dummySuccessProviderFb);
   withFeature('socialAuth');
   visit('/login');
 
   click('button:contains(Log in with Facebook)');
 
   andThen(() => {
-    assert.equal(currentURL(), '/');
+    assert.ok(/^index(_loading)?$/.test(currentRouteName()));
```
Comment (noslouch) [inline tests/acceptance/login-test.js]: I believe this is fixing something that should be handled by wrapping that click in an `andThen`.

File: yarn.lock [hunk 4]
```diff
@@ -3029,6 +3029,12 @@ ember-inflector@^1.9.2, ember-inflector@^1.9.4:
   dependencies:
     ember-cli-babel "^5.1.7"
 
+ember-keyboard@2.1.9:
```
Comment (noslouch) [inline yarn.lock]: Why is the lockfile updated with this commit?

Comment (walsh9) [inline yarn.lock]: This is when i upgraded mirage

**Pull request merged**